docs/queries.md

```diff
@@ -0,0 +1,170 @@
+
+{: .warn }
+A subscribe query will **not** subscribe the returned `Doc` instances. If you want the documents themselves to *also* receive updates, you will need to subscribe them manually using [`doc.subscribe()`]({{ site.baseurl }}{% link api/doc.md %}#subscribe).
+
+Subscribe queries can be created similarly to fetch queries, but you may also be interested in [other events]({{ site.baseurl }}{% link api/query.md %}#events):
+
+```js
+const query = connection.createSubscribeQuery('my-connection', {userId: 1})
+query.on('ready', () => {
+  // The initial results are available in query.results
+})
+query.on('changed', () => {
+  // This is a catch-all event that is fired when further changes are made.
+  // It is called just after the 'insert', 'move', and 'remove' events.
+})
+```
+
+## Performance
+
+Arbitrary queries are not necessarily performant out-of-the-box. As with all database queries, some steps should be taken to keep queries fast.
+
+### Indexing
+
+As with all database queries, appropriate indexes should be set up to expedite common queries. The exact details of this will vary depending on the underlying database.
+
+### Paging
+
+If a query can potentially return a large number of results, you may want to consider limiting the number of results that can be returned, again similarly to how a "traditional" query might be altered.
+
+For example, a `sharedb-mongo` limit might look like this:
+
+```js
+const query = connection.createSubscribeQuery('my-connection', {userId: 1, $skip: 10, $limit: 10})
+```
+
+{: .info }
+`sharedb-mongo` [queries](https://github.com/share/sharedb-mongo#queries) are not quite the same as MongoDB queries, and allow definition of some cursor functions directly in the query object.
+
+One way subscription queries act differently to a direct database query is that pages will automatically be updated. For example, consider this collection:
+
+```json
+{"userId": 1, "value": 5}
+{"userId": 1, "value": 7}
+{"userId": 1, "value": 3}
+{"userId": 1, "value": 1}
+{"userId": 2, "value": 2}
+```
+
+Let's run a limited query:
+
+```js
+const q = {userId: 1, $sort: {value: 1}, $limit: 3}
+const query = connection.createSubscribeQuery('values', q)
+query.on('ready', () => {
+  // query.results looks like this:
+  // [
+  //   {userId: 1, value: 1},
+  //   {userId: 1, value: 3},
+  //   {userId: 1, value: 5},
+  // ]
+})
+```
+
+Now let's insert a new document and listen for the change:
+
+```js
+const newDoc = connection.get('values', id)
+newDoc.create({userId: 1, value: 0})
+
+query.on('changed', () => {
+  // query.results has been updated:
+  // [
+  //   {userId: 1: value: 0},
+  //   {userId: 1: value: 1},
+  //   {userId: 1: value: 3},
+  // ]
+})
+```
+
```

Comment:
Feels like it would be a nice place to add some more information about how it works - I might be understanding it wrong but here is my shot:

```suggestion

The `query.results` contain the first 3 values from the query in sorted order, and not the 3 values that were returned initially. 

```